api/src/main/java/openconsensus/trace/propagation/BinaryFormat.java

```diff
@@ -80,10 +76,10 @@
    * Parses the {@link SpanContext} from a byte array using the binary format.
    *
    * @param bytes a binary encoded buffer from which the {@code SpanContext} will be parsed.
-   * @return the parsed {@code SpanContext}.
+   * @return the parsed {@code SpanContext} or {@code SpanContext.INVALID} if it could not be
```

Comment:
We said that we don't want to mention SpanContext.INVALID here.

Reply:
Updated this ;)